adagucserverEC/CXMLGen.cpp

```diff
@@ -215,7 +215,8 @@ int CXMLGen::getWMS_1_1_1_Capabilities(CT::string *XMLDoc, std::vector<MetadataL
             addErrorInXMLForMisconfiguredLayer(XMLDoc, layer);
           }
           if (layer->hasError == 0) {
-            XMLDoc->printconcat("<Layer queryable=\"%d\" opaque=\"1\" cascaded=\"%d\">\n", layer->layerMetadata.isQueryable, layer->dataSource->dLayerType == CConfigReaderLayerTypeCascaded ? 1 : 0);
+            XMLDoc->printconcat("<Layer queryable=\"%d\" opaque=\"1\" cascaded=\"%d\">\n", layer->layerMetadata.isQueryable,
```

Comment:
Would it be possible to derive isQueryable in the layerMetadata section? Then other code can rely on the same checks too.

Reply:
Note: As we agreed last week, this is not applicable (you had interpreted this line wrong when you wrote this comment)